packages/jest-jasmine2/src/index.js

```diff
@@ -25,10 +25,17 @@ const JASMINE = require.resolve('./jasmine/jasmine_light.js');
 async function jasmine2(
   globalConfig: GlobalConfig,
   config: ProjectConfig,
-  environment: Environment,
+  environment: ?Environment,
```

Comment:
woah, let's not do this please. For example by renaming the `environment` variable below?

Reply:
We have to; I need to kill all environment reference.